buildenv/jenkins/common/build

```diff
@@ -195,26 +196,50 @@ def checkout_pullrequest() {
             if (omr_upstream) {
                 sh "git config remote.origin.url https://github.com/eclipse/omr.git"
             }
-            checkout_pullrequest(OMR_PR)
+            checkout_pullrequest(OMR_PR,'omr')
         }
     }
 }
 
-def checkout_pullrequest(ID) {
+def checkout_pullrequest(ID,repo) {
     // Determine if we are checking out a PR or a branch
+    def JSONFILE = ""
+    def base1 = ""
+	def URL = "https://api.github.com/repos/eclipse/${repo}/pulls/${ID}"
+	withCredentials([usernamePassword(credentialsId: 'b6987280-6402-458f-bdd6-7affc2e360d4', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+ 		JSONFILE = sh returnStdout: true, script: "curl -u ${USER} --pass  ${PASS} ${URL}"
+	}
+	def slurper1 = new groovy.json.JsonSlurper()
+	def result1 = slurper1.parseText(JSONFILE)
+	base1 = result1.base.ref  
```

Comment:
Uppercase variable names. And perhaps a more descriptive variable name?

Reply:
fixed